packs/metaweather/README.md

```diff
@@ -0,0 +1,8 @@
+# MetaWeather Integration Pack
+
+Pack providing integration with MetaWeather API.
+
+## Actions
+
+* ``get_location_results`` - Retrieve a list of locations given a search query. Ex. ZIP code, Country/City/State, etc.
```

Comment:
Besides that, LGTM.

What else needs to be done to take this across the finish line?